src/java.ts

```diff
@@ -121,6 +120,10 @@ export async function handleJREProvisioning(
     latestJREData.filename + UNARCHIVE_SUFFIX,
   );
 
+  log(LogLevel.DEBUG, 'Detecting proxy');
+  const proxyUrl = getProxyUrl(properties);
+  log(LogLevel.DEBUG, proxyUrl ? `Proxy detected: ${proxyUrl}` : 'No proxy detected');
```

Comment:
you could move these logs into the proxy.ts when the decision is made, IMO it keeps the log closer to the logic (and removed conditional logs)

I realize I need to fix the logs for `supportsJREProvisioning` where I did the same thing 😓